installer/templates/phx_single/config/config.exs

```diff
@@ -24,6 +24,9 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:request_id]
 
+# Use Jason for JSON parsing in Phoenix
+config :phoenix, :format_encoders, json: Jason
```

Comment:
We also need to generate the configuration for Ecto:

    config :ecto, :json_library, Jason

Reply:
Is this deprecated? Should we generate `config :#{db_adapter}, :json_library, Jason` instead?